doc/sphinxext/math_symbol_table.py

```diff
@@ -6,17 +6,16 @@
 symbols = [
     ["Lower-case Greek",
      6,
-     r"""\alpha \beta \gamma \chi \delta \epsilon \eta \iota \kappa
-         \lambda \mu \nu \omega \phi \pi \psi \rho \sigma \tau \theta
-         \upsilon \xi \zeta \digamma \varepsilon \varkappa \varphi
-         \varpi \varrho \varsigma \vartheta""".split()],
+     r"""\alpha \beta \gamma \delta \epsilon \zeta \eta \theta \iota \kappa \lambda \mu
+         \nu \xi \pi \rho \sigma \tau \upsilon \phi  \psi  \chi \omega \digamma
```

Comment:
```suggestion
         \nu \xi \pi \rho \sigma \tau \upsilon \phi \chi \psi \omega \digamma
```

wiki says "..., Φ φ, Χ χ, Ψ ψ, ..."

https://en.wikipedia.org/wiki/Greek_alphabet